test/Microsoft.AspNetCore.Mvc.Razor.Test/RazorPageTest.cs

```diff
@@ -1188,7 +1188,7 @@ public static TheoryData WriteAttributeData
             return view.Object;
         }
 
-        private static ViewContext CreateViewContext(TextWriter writer = null)
+        private static ViewContext CreateViewContext(TextWriter writer = null, string viewPath = null)
```

Comment:
Shouldn't need to make changes here. `RazorView.Path` is almost-never referenced in product code (your incorrect update aside).